src/ResourceManager/AnalysisServices/ChangeLog.md

```diff
@@ -18,6 +18,8 @@
         - Additional information about change #1
 -->
 ## Current Release
+* Added State property in additional to ProvisioningState
```

Comment:
You need to be more specific about this change. Which cmdlets will be affected, how would the user use the new State property.  How is it different than Provinsioning State?